sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -186,11 +186,13 @@ task integrationTest(type: Test, dependsOn: processTestResources) {
   def gcpProject = project.findProperty('gcpProject') ?: 'apache-beam-testing'
   def gcpTempRoot = project.findProperty('gcpTempRoot') ?: 'gs://temp-storage-for-end-to-end-tests'
   def firestoreDb = project.findProperty('firestoreDb') ?: 'firestoredb'
+  def host = project.findProperty('host') ?: 'batch-firestore.googleapis.com:443'
```

Comment:
Consider leaving this unset by default (it would catch the bugs below where we assume it is set).

Reply:
Keeping default.  Otherwise, the default value is not provided.